DOL.WHD.Section14c.Web/src/modules/routes.js

```diff
@@ -0,0 +1,171 @@
+        template: function(params) {
+          return (
+            '<form-section><section-employer></section-employer></form-section>'
+          );
+        },
+        reloadOnSearch: false,
+        access: ROUTE_USER,
+        label: 'Section: Employer',
+        parent:'/section/app-info'
+      })
+      .when('/section/wage-data', {
+        template: function(params) {
+          return (
+            '<form-section><section-wage-data></section-wage-data></form-section>'
+          );
+        },
+        reloadOnSearch: false,
+        access: ROUTE_USER,
+        label: 'Section: Wage Data',
+        parent:'/section/employer'
+      })
+      .when('/section/work-sites', {
+        template: function(params) {
+          return (
+            '<form-section><section-work-sites></section-work-sites></form-section>'
+          );
+        },
+        reloadOnSearch: false,
+        access: ROUTE_USER,
+        label: 'Section: Work Sites & Employees',
+        parent:'/section/wage-data'
+      })
+      .when('/section/wioa', {
+        template: function(params) {
+          return (
+            '<form-section><section-wioa></section-wioa></form-section>'
+          );
+        },
+        reloadOnSearch: false,
+        access: ROUTE_USER,
+        label: 'Section: WIOA',
+        parent:'/section/work-sites'
+      })
+      .when('/section/review', {
+        template: function(params) {
+          return (
+            '<form-section><section-review></section-review></form-section>'
+          );
+        },
+        reloadOnSearch: false,
+        access: ROUTE_USER,
+        label: 'Section: Review',
+        parent:'/section/wioa'
+      })
+      .when('/admin', {
+        controller: 'adminDashboardController',
+        template: require('./pages/adminDashboardTemplate.html'),
+        access: ROUTE_ADMIN
+      })
+      .when('/admin/users', {
+        controller: 'userManagementPageController',
+        template: require('./pages/userManagementPageTemplate.html'),
+        access: ROUTE_ADMIN
+      })
+      .when('/admin/:app_id', {
+        redirectTo: function(params) {
+          return '/admin/' + params.app_id + '/section/summary';
+        },
+        access: ROUTE_ADMIN
+      })
+      .when('/admin/:app_id/section/:section_id', {
+        template: function(params) {
+          return (
+            '<admin-review appid=' +
+            params.app_id +
+            '><section-admin-' +
+            params.section_id +
+            ' item-id=' +
+            (params.item_id || '') +
+            '></section-admin-' +
+            params.section_id +
+            '></admin-review>'
+          );
+        },
+        reloadOnSearch: false,
+        access: ROUTE_ADMIN
+      })
+      .when('/v4/hello', { template: '<hello-world></hello-world>' })
+      .when('/v4/ui-library', { template: '<ui-library></ui-library>' })
+      .otherwise({
+        redirectTo: '/'
+      });
+  });
+}
```

Comment:
For the sake of the coverage requirements, this file will need a test.  The way you wrote it should make it really easy to test.  Something like this might work:

```js
var routes = require('./routes');

var whenCallCount = 0;
var otherwiseCallCount = 0;
var routeProvider = {
  when: function() {
    whenCallCount++;
  },
  otherwise: function() {
    otherwiseCallCount++;
  }
};

var appCallCount = 0;
var app = {
  function(fn) {
    appCallCount++;
    expect(typeof(fn)).toEqual('function');
    fn(routeProvider);
  }
};

routes(app);
expect(appCallCount).toEqual(1);
expect(whenCallCount).toBeGreaterThan(0);
expect(otherwiseCallCount).toEqual(1);
```

Or if you wanted to add [sinon](http://sinonjs.org/) to the project, you could use that for mocking instead of writing your own mocks, and that'd be pretty dandy too.